google/cloud/aiplatform/models.py

```diff
@@ -234,8 +260,22 @@ def _create(
             project=project, location=location
         )
 
+        # Use provided encryption key name or else use one from global config
+        kms_key_name = (
```

Comment:
It looks like this block wasn't updated to use the global config. Also this should be set in `create` instead of this method which can be executed asynchronously.

Reply:
Done